modules/1_vpc_support/variables.tf

```diff
@@ -60,6 +60,7 @@ variable "vpc_supp_public_ip" {
 }
 
 variable "powervs_machine_cidr" {}
-variable "override_transit_gateway_name" {}
+variable "setup_transit_gateway" {}
```

Comment:
We could make this explicitly a boolean

Reply:
It is defined as Boolean in main variables.tf